--- a/libqtile/widget/__init__.py	2020-04-14 19:28:31.000000000 +0600
+++ b/libqtile/widget/__init__.py	2020-09-14 16:24:07.172778477 +0600
@@ -59,6 +59,8 @@
 safe_import("maildir", "Maildir")
 safe_import("notify", "Notify")
 safe_import("sensors", "ThermalSensor")
+safe_import("hddsensors", "HDThermalSensor")
+safe_import("chsensors", "ChThermalSensor")
 safe_import("sep", "Sep")
 safe_import("she", "She")
 safe_import("spacer", "Spacer")
